src/csharp/Microsoft.Spark/Interop/Internal/Java/Util/ArrayList.cs

```diff
@@ -23,5 +23,15 @@ internal ArrayList(IJvmBridge jvm)
         }
 
         JvmObjectReference IJvmObjectReferenceProvider.Reference => _jvmObject;
+
+        public void Add(object element)
+        {
+            _jvmObject.Invoke("add", element);
+        }
+
+        public void Remove(int index)
+        {
+            _jvmObject.Invoke("remove", index);
+        }
```

Comment:
You can also expose `public void AddRange` to make the conversion easier

Reply:
I was not able to find `addRange()` in `java.util.ArrayList`. I found an `addAll()` that takes in a `java.util.Collection` type as argument. Unless you mean the `AddRange()` in c# in which case I am missing something, please clarify if that's the case. Thanks!

Reply:
You can add `AddRange()` which calls `Add()` in a loop.